app/mailers/daily_news_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class DailyNewsMailer < ApplicationMailer
+  def daily_email(user, lots_amount)
+    @user = user
```

Comment:
Why do you need instance variables for lots_amount here?
Use can use Lots.count inside the view.